frontend/src/app/shared/components/enterprise-banner/enterprise-banner.component.sass

```diff
@@ -8,8 +8,13 @@
     margin-top: 10px
 
   &--description
-    margin-bottom: 10px
     display: block
+    font-size: 14px
+    font-weight: 400
+    line-height: 16px
```

Comment:
shouldn't we use here the correct include of the spot font blocks? the one with `font-size: 0.875rem`?